enhancements/ingress/gateway-api-crd-life-cycle-management.md

```diff
@@ -147,7 +148,41 @@ from OpenShift 4.19.0.
 
 ### Dead fields
 
-_TBD_: Describe the issue and how we will address it in 4.19.
+We define "dead fields" as the situation where an implementation that uses CRDs
+(such as our Gateway API implementation) is technically _compatible_ with a
+newer installed version of the CRDs, but fields in that new version are inert
+when specified and the user would have insufficient feedback to be aware of
+this. In the worse cases, the user may believe something critical is configured
+on the running configuration, but it is not and the remaining configuration is
+erroneously deployed anyway.
+
+A primary concern here would be when a new version of a CRD provides some new
+kind of security sensitive field. It may not always be intuitive for a user to
+know by way of component awareness or documentation that such a field may be
+_present_, but wont actually be accomodated by the implementation. For example,
+if a newer version of the API provides a new authentication related field and
+the user specifies that not knowing the implementation doesn't support it, we
+would want to protect against deploying the remaining specification even were
+it "technically functional", so that we're not exposing resources without
+authentication that were intended to require authentication.
+
+In 4.19.0 we will address this by restricting upgrades of the Gateway API CRDs
+only to newer versions that have been vetted by us explicitly and are known to
+be free from unsafe dead fields with the corresponding version of our
+first-party implementation. This will mean that we will maintain a documented,
+tested and enforced range of Gateway API CRDs versions for any release of OCP.
+Initially this will provide a somewhat restrictive range of CRD versions
+available to users of third-party options, and we anticipate over time there may
+be a need to expand it.
```

Comment:
My understanding from our discussions was that we had decided not to allow a range but instead to pin to a single version.  That version can be the one corresponding to the Istio version that we ship, or it can be some newer version as long as we have verified that that newer version combined with the Istio we ship passes the upstream conformance tests and our downstream end-to-end tests.